index.html

```diff
@@ -11037,6 +11037,7 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 				&lt;input id="startTime" type="text" aria-errormessage="msgID" aria-invalid="true" value="11:30 PM" &gt;
 				&lt;span id="msgID" aria-live="assertive"&gt;&lt;span style="visibility:visible"&gt;Invalid time: the time must be between 9:00 AM and 5:00 PM&lt;/span&gt;&lt;/span&gt;
 				</pre>
+				<p><code>aria-live="assertive"</code> is used in this example in favor of <code>aria-live="polite"</code> to ensure the error message is read before the user is done interacting with the field. Otherwise, the user may confuse the message with that of the next field.</p>
```

Comment:
the content that precedes the code example refers to the form controls as "text inputs", but this new content refers to them as "fields".

I suggest we stick with the previous use of "text input"